mobile/src/screens/marketplace.js

```diff
@@ -473,6 +476,36 @@ class MarketplaceScreen extends PureComponent {
     )
   }
 
+  injectDeviceFingerprint = async () => {
+    const screenResolution = `${Math.round(
+      Dimensions.get('window').width
+    )}x${Math.round(Dimensions.get('window').height)}`
+
+    const fingerprint = {
+      deviceId: DeviceInfo.getDeviceId(),
+      brand: DeviceInfo.getBrand(),
+      deviceType: DeviceInfo.getDeviceType(),
+      osName: DeviceInfo.getSystemName(),
+      osVersion: DeviceInfo.getSystemVersion(),
+      userAgent: await DeviceInfo.getUserAgent(),
+      uuid: DeviceInfo.getUniqueId(),
+      ipAddress: await DeviceInfo.getIpAddress(),
+      screenResolution
```

Comment:
Any chance we can get the language or locale? Does not look like the react-native library exposes that. But maybe from the webview using something like navigator.language? though it may not be supported universally...

Reply:
Let me look into that.

Reply:
I have added it @franckc